lib/render-html.test.js

```diff
@@ -5,9 +5,6 @@ describe('renderHTML', () => {
     const originalRequireResolve = require.resolve;
     const mockThemePath = 'mock/path/to/jsonresume-theme-even';
     require.resolve = (...args) => {
```

Comment:
While writing the new tests, I found that this mock of resolve is not called.
So tests are running against the actual file system.
Possibly the mock can be removed.  Or maybe it needs to be fixed.

Reply:
The PR comment is an implicit question about what the correct action is.
Any TODO would come from the answer to the question.
And I'm not sure that in code TODOs is the way this project manages future code changes.  They may prefer creating a GitHub issue.